opal/runtime/opal_progress.c

```diff
@@ -480,3 +485,28 @@ int opal_progress_unregister (opal_progress_callback_t cb)
 
     return ret;
 }
+
+void opal_progress_handle_hangup(opal_progress_hangup_callback_fn_t cbfunc,
+                                 void *cbdata)
+{
+    char prefix[100 + OPAL_MAXHOSTNAMELEN];
+    FILE *stream = stderr;
```

Comment:
Instead of hardcoding `stderr` could we harness the `opal_stacktrace_output_filename` MCA parameter (`-mca opal_stacktrace_output`) to allow for some flexibility in the destination of the stack output (so we can choose a file, for example). There is some logic in `opal/util/stacktrace.c` to process that MCA parameter. Maybe we can move that logic to a more commonly accessible area.